src/templates/pycontw-2020/ext/live.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'contents/_base.html' %}
+
+{% load i18n compress %}
+
+{% block title %}{% trans 'Live broadcast' %}{% endblock %}
+
+{% block body_class %}{{ block.super }} live-page{% endblock body_class %}
+
+{% compress js %}
+{% block scripts %}
+{{ block.super }}
+<script>
+var next = document.querySelector("[name='next']");
```

Comment:
I should have express more on the situation why this code is needed.

`[name='next']` is located at `src/templates/pycontw-2020/_includes/top_nav.html`

```html
            <form class="navbar-lang-switch" method="post" action="{% url 'set_language' %}">
                {% csrf_token %}
                <input type="hidden" name="next" value="{{ request|language_free_path }}">

                {% get_language_info_list for LANGUAGES as languages %}
                {% get_current_language as LANGUAGE_CODE %}
                {% get_language_info for LANGUAGE_CODE as current_language %}

                {% for language in languages %}
                <button type="submit" name="language" value="{{ language.code }}" {% if language.code == LANGUAGE_CODE %}disabled="disabled"{% endif %}>
                    {{ language.code.upper|slice:'0:2' }}
                </button>
                {% endfor %}

            </form>
```

The rendered code would be something like the following
```html
            <form class="navbar-lang-switch" method="post" action="/2020/set-language/">
                <input type="hidden" name="csrfmiddlewaretoken" value="SOMECSRFTOKEN">
                <input type="hidden" name="next" value="/2020/events/talks/">
                <button type="submit" name="language" value="zh-hant" disabled="disabled">
                    ZH
                </button>
                <button type="submit" name="language" value="en-us" >
                    EN
                </button>
            </form>
```

This particular page needs the `token` query parameter. But in current implementation, if you switch language to another, it'll switch the language but all query parameters will be gone. So this is merely a hack to inject the token into the next parameter in the switch language function, so that the token will be preserved after the language redirect.

Reply:
how about leaving a simple, brief comment here for explaining the purpose of this snippet?